usr/share/rear/prep/GNU/Linux/305_include_s390_tools.sh

```diff
@@ -0,0 +1,29 @@
+#
+#  s390 zIPL boot loader and grubby for configuring boot loader`
+
+test -d $VAR_DIR/recovery || mkdir -p $VAR_DIR/recovery
+
+local bootdir="$( echo -n /boot/ )"
+test -d "$bootdir" || $bootdir='/boot/'
+
+# cf. https://github.com/rear/rear/issues/2137
+# findmnt is used the same as grub-probe to find the device where /boot is mounted
+# example
+# findmnt -no SOURCE --target /boot
+# --> /dev/dasda1
+#
+# sles will need to be looked at:
+# findmnt returns --> /dev/dasda3[/@/.snapshots/1/snapshot]
+# it's possible that sles can use 300_include_grub_tools.sh instead of this file
+if has_binary findmnt ; then
+    echo 'run findmnt'
+    findmnt -no SOURCE --target $bootdir >$VAR_DIR/recovery/bootdisk 2>/dev/null || return 0
```

Comment:
FYI:
because STDERR is redirected to the ReaR log file
you can omit `2>/dev/null` if you like to get error messages
in the log file where they help to identify the root cause of a problem.
On the other hand if a program is needelssly verbose you may
use `2>/dev/null` or `1>/dev/null` or `&>/dev/null` as appropriate
to avoid that the ReaR log file is "polluted" with useless messages.
In general - from my point of view - I prefer to have rather more than less
messages in the ReaR log file because more messages help to identify
the root cause of a problem - in particular for me when I must look
at a ReaR log file from another user and then I must imagine only
from the log messages what was actually going on on the user's system.

Reply:
I copied this from 300_include_grub_tools.sh and changed grub-probe to findmnt

I will remove the _stderr redirect to null_,  would you like me to remove it from 300_include_grub_tools.sh  also?

```
if has_binary grub-probe ; then
    grub-probe -t device $grubdir >$VAR_DIR/recovery/bootdisk 2>/dev/null || return 0
elif has_binary grub2-probe ; then
    grub2-probe -t device $grubdir >$VAR_DIR/recovery/bootdisk 2>/dev/null || return 0
fi
```

As an aside grub2-probe work on sles